dom.bs

```diff
@@ -6399,74 +6574,82 @@ null), and boolean <var>synchronousCustomElements</var> (default false):
 
   <ol>
    <li>
-    <p>If <var>synchronousCustomElements</var> is true, then run these steps while catching any
-    exceptions:
+    <p>If <var>synchronousCustomElements</var> is true:
 
     <ol>
      <li><p>Let <var>C</var> be <var>definition</var>'s
      <a for="custom element definition">constructor</a>.
 
-     <li><p>Set <var>result</var> to the result of <a>constructing</a> <var>C</var>, with no
-     arguments.
-
-     <li><p>Assert: <var>result</var>'s <a for=Element>custom element state</a> and
-     <a for=Element>custom element definition</a> are initialized.
+     <li><p><a for=map>Set</a> the <a>surrounding agent</a>'s
```

Comment:
It's very hard to follow the diff here inside "create an element", so for any other reviews, I think the main things are:

- Refactoring to use a "creating an element internal" helper, which takes the registry as an argument.
- A bunch of steps nested one more level under "Run these steps while catching any exceptions", ending with the added "Set result’s custom element registry to registry."

Basically searching for "registry" within the steps shows the interesting stuff.

And I can't find anything wrong :)